images_to_samples.py

```diff
@@ -225,8 +230,9 @@ def samples_preparation(in_img_array,
                     padding = pad_diff(target_row, target_col, sample_size,
                                        sample_size)  # array, actual height, actual width, desired size
                     target = pad(target, padding, fill=dontcare)
-                u, count = np.unique(target, return_counts=True)
-                target_background_percent = round(count[0] / np.sum(count) * 100 if 0 in u else 0, 1)
+                backgr_ct = np.sum(target == 0)
+                backgr_ct += np.sum(target == dontcare)
+                target_background_percent = round(backgr_ct / target.size * 100, 1)
 
```

Comment:
Suggestion/thoughts: different from target_background percent/min_annot_percent I think it would be 'a nice to have' if we are able to keep 20/30%(even less %5) of samples containing all background while disposing of 80/70%

Reply:
definitely. Could you open a "feature enhancement" issue?